.dockerignore

```diff
@@ -0,0 +1,4 @@
+venv
+.venv
+Dockerfile
+docker-compose.yaml
```

Comment:
The `docker-compose.yaml` file should not be ignored in the `.dockerignore` file, as it is necessary for defining and running multi-container Docker applications.